database/sql/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -100,6 +100,8 @@ class SlickDatabase(override val originalDatabaseConfig: Config) extends SqlData
     actionThreadPool, database.executor.executionContext.reportFailure
   )
 
+  protected[this] lazy val insertBatchSize = databaseConfig.as[Option[Int]]("db.insertBatchSize").getOrElse(2000)
```

Comment:
ah, `database` it is. Which also means I can not feel bad about reverting to our more typical `insert-batch-size`